src/Colopl/Spanner/Concerns/ManagesTransactions.php

```diff
@@ -158,7 +161,14 @@ protected function performRollBack($toLevel)
 
         if ($this->currentTransaction !== null) {
             if ($this->currentTransaction->state() === Transaction::STATE_ACTIVE) {
-                $this->currentTransaction->rollBack();
+                try {
+                    $this->currentTransaction->rollBack();
+                } catch (NotFoundException $e) {
+                    // ignore session not found error
+                    if (empty($this->getSessionNotFoundMode()) || !$this->causedBySessionNotFound($e)) {
```

Comment:
I think it's safe to remove `empty($this->getSessionNotFoundMode())` since it will always contain a value.

Reply:
I thought to allow disabling mode by simple 
```
   	        'sessionNotFoundErrorMode' => false,
```
I am going to redo it to 
   	        'sessionNotFoundErrorMode' => 'THROW_EXCEPTION',

Reply:
done in recent commits